renovate.json

```diff
@@ -1,23 +1,27 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base"
-  ],
+  "extends": ["config:base"],
   "schedule": ["every weekend"],
```

Comment:
Can we move this to monthly, except for cosign and its dependencies?

Reply:
I think this will lighten the burden significantly already. We can move to monthly updates if it's still too much.